test/test0032/enum.c

```diff
@@ -0,0 +1,121 @@
+/* Copyright (C) 2022 Andrius Štikonas
```

Comment:
This file is not written by me :). Do you want to update copyright statement?

Reply:
Force pushed to update copyright.